tests/spec/examples.py

```diff
@@ -403,7 +403,7 @@ def it_displays_correctly_when_the_service_is_up_json(self, in_example_dir):
                 'state': 'ready',
             },
         }
-        assert stderr == b''
+        assert stderr == ''
```

Comment:
It doesn't require decoding in almost every case, it just requires you to compare to bytes instead of unicode (which is the logical operation anyway, since these these are byte streams).

From the above broken tests, I'd argue the current API is surprising.